# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sox/hotfix.patch.musl
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/formats.c.orig	2013-02-17 12:19:38.318344596 +0000
+++ ./src/formats.c	2013-02-17 12:50:31.560549872 +0000
@@ -417,7 +417,7 @@
   /* To fix this #error, either simply remove the #error line and live without
    * file-type detection with pipes, or add support for your compiler in the
    * lines above.  Test with cat monkey.wav | ./sox --info - */
-  #error FIX NEEDED HERE
+//  #error FIX NEEDED HERE
   #define NO_REWIND_PIPE
   (void)fp;
 #endif
